boards/arm/cc3235sf_launchxl/pinmux.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * pinmux.c
+ *
+ * configure the device pins for different peripheral signals
```

Comment:
is there a reason this isn't apache licensed?

Reply:
@galak I think it should be fine, but given it's another copy of a file that already exists, I didn't change the license. If you think it makes sense to change it to Apache I can modify it.